Docs/source/usage/parameters.rst

```diff
@@ -125,7 +125,10 @@ Setting up the field mesh
 * ``geometry.is_periodic`` (`2 integers in 2D`, `3 integers in 3D`)
     Whether the boundary conditions are periodic, in each direction.
 
-    For each direction, use 1 for periodic conditions, 0 otherwise.
+    For each direction, use 1 for periodic conditions, 0 otherwise. If
```

Comment:
Can you move this documentation to the place where the boundary.field_lo and hi are described?